README.md

```diff
@@ -5,17 +5,58 @@
 [![Dependency Status](https://david-dm.org/silvermine/eslint-plugin-silvermine.png)](https://david-dm.org/silvermine/eslint-plugin-silvermine)
 [![Dev Dependency Status](https://david-dm.org/silvermine/eslint-plugin-silvermine/dev-status.png)](https://david-dm.org/silvermine/eslint-plugin-silvermine#info=devDependencies&view=table)
 
-
 ## What?
 
-Shareable [eslint](http://eslint.org/) plugins that are used by our shareable config rules. See [eslint-config-silvermine](https://github.com/silvermine/eslint-config-silvermine/) for more details.
-
+Shareable [eslint](http://eslint.org/) plugins that are used by our shareable config rules.
+See [eslint-config-silvermine](https://github.com/silvermine/eslint-config-silvermine/) for more details.
 
 ## Why?
 
 Because we need it. Whitespace errors are evil. As are the other hundreds of types of errors this protects us from.
 
+## Installation
+
+You'll first need to install [ESLint](http://eslint.org):
+
+```
+$ npm install eslint --save-dev
+```
+
+Next, install `eslint-plugin-silvermine`:
+
+```
+$ npm install eslint-plugin-silvermine --save-dev
+```
+
+**Note:** If you installed ESLint globally (using the `-g` flag) then you must also install `eslint-plugin-silvermine` globally.
+
+## Usage
+
+Add `silvermine` to the plugins section of your `.eslintrc` configuration file. You can omit the `eslint-plugin-` prefix:
+
+```json
+{
+    "plugins": [
+        "silvermine"
+    ]
+}
+```
+
+
+Then configure the rules you want to use under the rules section.
+
+```json
+{
+    "rules": {
+        "silvermine/fluent-chaining": 2
+    }
+}
+```
+
+## Supported Rules
 
-## License
+fluent-chaining
```

Comment:
Let's make this a bulleted list and a link to the documentation on it. For example:

```
   * [fluent-chaining](docs/rules/fluent-chaining.md)
```